readme.md

```diff
@@ -21,9 +27,9 @@ Dependencies are defined in a `deps.json` file, e.g.
 ## How to use
 
 1. Create a `deps.json`.
-2. Run `courier` from your shell.
+2. Run `courier`.
 3. Check-in `deps.json` and `pins.json` (created by courier)
-4. To obtain the exact same dependencies later, run `courier -reproduce`.
+4. To obtain the exact same dependencies later, run `courier --reproduce`.
```

Comment:
You have changed the comments to `--reproduce`, however if you run `courier -h` what you see in the output is `-reproduce`. 
That being said, running it with either works.